src/core/public/application/types.ts

```diff
@@ -44,6 +45,12 @@ export interface AppBase {
    */
   title: string;
 
+  /**
+   * The category the app lives in
+   * Default categories defined in {@link DEFAULT_APP_CATEGORIES}
```

Comment:
NIT: link to constant doesn't work. The generated doc is `The category the app lives in Default categories defined in  `. Should probably replace with a link to `AppCategory`